scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
Hmm, so do we know _which_ optional dependencies are needed by the installers? Rather than re-running the whole of `make electron-develop`, is there maybe some way to only install the optional dependencies needed by the installers?

Hmmmmm, thinking about it I guess the "problem" is that we have the dependencies (with some of them being optional) required by Etcher itself, and we also have the dependencies (with some of them being optional) required to create the Etcher installers; and from a conceptual point of view these are two entirely separate sets of dependencies, and yet from a practical point of view they're both stored in the same `package.json` / `npm-shrinkwrap.json` file? (have I understood that correctly?)
Hopefully this doesn't mean that we end up with modules only needed to create the Etcher installers, included inside Etcher's `.asar`?